linera-base/src/identifiers.rs

```diff
@@ -160,19 +205,47 @@ impl From<ApplicationId> for GenericApplicationId {
 }
 
 /// A unique identifier for an application bytecode.
+#[derive(WitLoad, WitStore, WitType)]
 #[cfg_attr(with_testing, derive(Default))]
 pub struct BytecodeId<Abi = (), Parameters = (), InitializationArgument = ()> {
     /// The message ID that published the bytecode.
     pub message_id: MessageId,
+    #[witty(skip)]
```

Comment:
I think this should be allowed, so I opened #1889. However, it might make sense to skip it in this case to simplify the generated WIT snippet (and consequently the types generated by `wit-bindgen`) :thinking:

Reply:
Ah, right, you'd probably get a useless field with unit type; good point.